tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm

```diff
@@ -500,15 +500,17 @@ sub _build_variants_frequency_data {
         'value' => 'yes',
         'checked' => 1,
         'field_class'   => [qw(_stt_yes _stt_allele)],
-      }), $fieldset->add_field({
-        'type' => 'checkbox',
-        'name' => 'failed',
-        'label' => $fd->{failed}->{label},
-        'helptip' => $fd->{failed}->{helptip},
-        'value' => 1,
-        'checked' => 0,
-        'field_class'   => [qw(_stt_yes _stt_allele)],
       })]
+    }) if (first { $_->{'value'} eq 'Homo_sapiens' } @$species);
+
+    $fieldset->add_field({
```

Comment:
This field is for Homo_sapiens only. Check screenshot.
![Screenshot 2020-10-07 at 10 37 00](https://user-images.githubusercontent.com/6347854/95314408-2ccb6100-0889-11eb-88b5-15c5ad40dffc.png)

Reply:
Yep thats correct, its been there before and it is specific to human only.